LSM: Update comment in Groove to reflect code changes

ifreund opened the pull request.

ifreund committed:
LSM: Update comment in Groove to reflect code changes

sentientwaffle reviewed (dismissed):
Thanks for the clarification!

ifreund commented:
@sentientwaffle I've just fixed a small typo and rebased onto main, but it looks like GitHub requires a fresh press of the Approval button before this can be merged if you don't mind :)
